server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -14,41 +14,110 @@
+    List<Mutation> mutations = new ArrayList<>();
+    if (StorageHandlerHelper.checkPersonInClubQuery(dbClient, userId, clubId)) {
+      throw new IllegalArgumentException(MembershipConstants.PERSON_ALREADY_IN_CLUB);
+    }
+    mutations.add(
+      Mutation.newInsertBuilder("Memberships")
+        .set("userId")
+        .to(userId)
+        .set("clubId")
+        .to(clubId)
+        .set("membershipType")
+        .to(MembershipConstants.MEMBER)
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build());
+      dbClient.write(mutations);
+  }
+
+  /**
+  * Deletes a membership from the database.
+  * This method checks if a person is already a member of a club. If the person is not a member, it will throw
+  * an exception. Otherwise, it deletes the row containing the user ID and club ID from the Memberships table.
+  *
+  * @param  dbClient    the database client 
+  * @param  userId      the user ID string used to query the membership table
+  * @param  clubId      the club ID string used to query the membership table
+  */
+  public static void deletePersonClubMembershipDml(DatabaseClient dbClient, String userId, String clubId) {
+    if (!StorageHandlerHelper.checkPersonInClubQuery(dbClient, userId, clubId)) {
+      throw new IllegalArgumentException(MembershipConstants.PERSON_NOT_IN_CLUB);
+    }
+    dbClient
+      .readWriteTransaction()
+      .run(
+          new TransactionCallable<Void>() {
+            @Override
+            public Void run(TransactionContext transaction) throws Exception {
+              String sql = "DELETE FROM Memberships WHERE userId = '"
+                    + userId + "' AND clubId = '" + clubId + "'";
+              long rowCount = transaction.executeUpdate(Statement.of(sql));
+              System.out.printf("%d record deleted.\n", rowCount);
+              return null;
+            }
+          });
+  }
+
   /**
-  * Returns a string containing information about a person queried from the database.
-  * This method formats a string with a user's ID, email, nickname, and pronouns.
+  * Creates and returns a {@link Person} with the info about a person queried from the database.
+  * This method builds a {@link Person} with a user's ID, email, nickname, and pronouns.
   * If the user has no pronouns (NULL), or the pronouns are empty, then "No pronouns"
   * is printed out instead.
   *
   * @param  dbClient    the database client 
   * @param  userId      the user ID string used to query and get a person's information
-  * @return personInfo  the formatted string containing the person information
+  * @return person      the Person object built containing the person information
   */
-  public static String getPersonQuery(DatabaseClient dbClient, String userId) {
-    String personInfo = "";
+  public static Person getPersonQuery(DatabaseClient dbClient, String userId) {
```

Comment:
Nit: This reads to me as getting the query that returns the person. I think just `getPerson` would be clearer. I mentioned previously that it might be good to split out some functions that just return a Mutation or list of Mutations, so those could be called `getPersonQuery` or `getPersonMutation`